src/include/stir/scatter/ScatterEstimation.h

```diff
@@ -5,6 +5,7 @@
 /*
     Copyright (C) 2018 - 2019 University of Hull
     Copyright (C) 2016,2020 University College London
+    Copyright (C) 2022 Nationcal Physical Laboratory
```

Comment:
I guess currently no changes in this file from you, so I'd remove this (and the line below), unless you will change it